src/components/Gauge.js

```diff
@@ -0,0 +1,263 @@
+      { startAngle: sixthSegment, endAngle: endOfSixthSegment, padAngle: 0.02 }];
+
+    return segments.map((point, i) => {
+      return (
+        <g
+          key={i}
+        >
+          <path
+            className={'arc-' + this.props.id}
+            d={this.state.standardArc(point)}
+            fill={this.props.colors[i]}
+            opacity={this.props.opacity}
+            ref={'arc-' + this.props.id + i}
+          />
+        </g>
+      );
+    });
+  },
+
+  _renderBaseArc () {
+    return (
+      <g>
+        <path d={this.state.baseArc()} fill={this.props.baseArcColor}></path>
+      </g>
+    );
+  },
+
+  _renderDataPoints () {
+    const dataPoints = this.props.dataPoints.map(dataPoint => {
+      return dataPoint.value;
+    });
+
+    return dataPoints.map((dataPoint, index) => {
+      const percentOfTotal = dataPoint / this.props.chartTotal;
+      const endAngle = (percentOfTotal * 270) - 135;
+
+      const dataPointArc = d3.svg.arc()
+        .outerRadius(this.state.radius - this.props.activeOffset)
+        .innerRadius(this.state.radius - this.props.arcWidth)
+        .startAngle(endAngle * (Math.PI / 180))
+        .endAngle(endAngle * (Math.PI / 180));
+
+      return (
+        <circle
+          cx='0'
+          cy='0'
+          fill={this.props.dataPointColors[index]}
+          key={index}
+          r={this.props.dataPointRadius}
+          transform={'translate(' + dataPointArc.centroid() + ')'}
+        />
+      );
+    });
+  },
+
+  _renderDataLabel () {
+    const styles = this.styles();
+
+    if (this.props.showDataLabel) {
+      if (this.props.children) {
+        return (
+          <div
+            className='mx-gauge-data'
+            onClick={this._handleClick}
+            style={styles.center}
+          >
+            {this.props.children}
+          </div>
+        );
+      } else {
+        const color = this.props.colors[0];
+        const text = this.props.defaultLabelText;
+        const value = this.props.formatter(this.props.defaultLabelValue);
+
+        return (
+          <div
+            className='mx-gauge-data'
+            onClick={this._handleClick}
+            style={styles.center}
+          >
+            <div className='mx-gauge-data-value' style={[styles.value, { color }]}>
+              {value}
+            </div>
+            <div className='mx-gauge-data-label' style={styles.label}>
+              {text}
+            </div>
+          </div>
+        );
+      }
+    } else {
+      return null;
+    }
+  },
+
+  render () {
+    const position = 'translate(' + this.props.width / 2 + ',' + this.props.height / 2 + ')';
+    const fontSize = Math.min(this.props.width, this.props.height) * 0.2;
+    const styles = this.styles();
+
+    return (
+      <div
+        className='mx-gauge'
+        style={[styles.component, this.props.style, { fontSize, height: this.props.height, width: this.props.width }]}
```

Comment:
If we're not using Radium for it's hover/focused/browser prefixing then we've decided to not use it in a component.  In stead, we've been using `Object.assign` to merge styles together like this.

```javascript
style={Object.assign({}, styles.component, this.props.style, { fontSize, height: this.props.height, width: this.props.width })}
```